DragaliaAPI.Database/Repositories/IUserDataRepository.cs

```diff
@@ -21,4 +21,7 @@ public interface IUserDataRepository : IBaseRepository
     Task UpdateCoin(long offset);
     Task<bool> CheckCoin(long quantity);
     Task GiveWyrmite(int quantity);
+    Task UpdateDewpoint(int quantity);
+    Task<bool> CheckDewpoint(int quantity);
+    Task SetDewpoint(int quantity);
```

Comment:
In what situation do you need to set the user's eldwater to an absolute value rather than +/- ing it by some value?

Reply:
I think this one was just for testing? To set the eldwater value to something that I know won't be enough